deploy/health_check/main.py

```diff
@@ -0,0 +1,71 @@
+import requests
+import yaml
+from flask import Flask
+from flask_cors import CORS
+
+app = Flask(__name__)
+CORS(app)
+
+# For 'health', true is healthy
+health = True
+CORE_URL = ''
+
+
+def check_core(core_url):
+    # Check if the stellar core is synced
+    global health
+    try:
+        r = requests.get(core_url+'/info')
+        r.raise_for_status()
+
+        health = (r.json()['info']['state'] == 'Synced!')
+    except:
+        health = False
+
+
+def check_horizon():
+    # Check if the ratio of success/failed requests is < 0.1
+    global health
+    try:
+        r = requests.get('http://horizon-kik.kininfrastructure.com/metrics')
+        r.raise_for_status()
+
+        total_requests = r.json()['requests.total']['1m.rate']
+        failed_requests = r.json()['requests.failed']['1m.rate']
```

Comment:
if you're fetcthing data dfrom `r.json()` more than once, cache it in a var before:
```python
data = r.json()
total_requests = ...
```